client/client.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+VARIANT_ENDPOINT = "http://rest.ensembl.org/vep/human/hgvs/{variant}" 
+
+class VariantAnnotationClient:
+    def __init__(self, variant_endpoint=VARIANT_ENDPOINT):
```

Comment:
Providing a default value that was a live server is probably not something I would usually do, but I figured it's fine for the purposes of this assignment.